src/components/skeletons/skeletons.tsx

```diff
@@ -1,74 +1,5 @@
-import React from "react";
 import styled from "styled-components";
 
-const LoginsSkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonLine />
-      <SkeletonBlock />
-      <SkeletonLine />
-      <SkeletonBlock />
-    </SkeletonBody>
-  );
-};
-
-const IdentityOverviewSkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonBlock />
-      <SkeletonLine />
-    </SkeletonBody>
-  );
-};
-
-const UpdateIdentitySkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonSection />
-      <SkeletonBlock />
-      <SkeletonLine />
-    </SkeletonBody>
-  );
-};
-
-const SecuritySkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonSection />
-    </SkeletonBody>
-  );
-};
-
-const TwoFASkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonBlock />
-    </SkeletonBody>
-  );
-};
-
-const BreadcrumbsSkeleton: React.FC = () => {
-  return (
-    <SkeletonBody>
-      <SkeletonTextLine fontSize={1.4} />
-    </SkeletonBody>
-  );
-};
-
-const SkeletonBody = styled.div`
-  background-color: ${({ theme }) => theme.colors.background};
-  height: auto;
-  width: auto;
-  overflow: hidden;
-  border-radius: ${({ theme }) => theme.radii[1]};
-
-  @keyframes loading {
-    100% {
-      transform: translateX(100%);
-    }
-  }
-`;
-
 const SkeletonAnimation = styled.div`
```

Comment:
Nothing there is used anymore? :o

Reply:
Nop, as we only load text now, or in special cases, we re-create a disabled `<BoxedLink />`.